portal/assets/js/global.js

```diff
@@ -68,4 +68,21 @@ document.addEventListener('DOMContentLoaded', function() {
     addMaildrop.parentNode.insertBefore(inputDropClone, addMaildrop);
   });
 
+  // FIXME - I don't understand what this do ...
+  // This looks kinda hackish :|
+  if(window.location != window.parent.location) {
+    // Set class to body to show we're in overlay
+    document.body.classList.add('overlay');
+      let userContainer = document.querySelector('a.user-container');
+      userContainer.classList.replace('user-container-info', 'user-container-edit');
+      userContainer.setAttribute('href', userContainer
+          .getAttribute('href')
+          .replace('edit.html', ''));
+      userContainer.addEventListener('click', function(e) {
```

Comment:
This prevents the link's default behavior (which is navigate inside the iframe) and navigates to the page on the main page instead of inside the iframe.